tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js

```diff
@@ -143,39 +162,54 @@ testsToRun.forEach(test => {
                     const key = `${prefix}ver-single-master`;
                     cloudServer.setKey(key);
                     cloud.setKey(`${srcBucket}/${key}`);
+                    let versionId = null;
                     series([
-                        next => cloudServer.putObject(Buffer.from(key), next),
-                        next => checkTransition(toLoc, cloudServer, cloud, null, next),
+                        next => cloudServer.putObject(Buffer.from(key), (err, data) => {
+                            if (data) {
+                                versionId = data.VersionId;
+                            }
+                            next(err);
+                        }),
+                        next => checkTransition(toLoc, cloudServer, cloud, versionId, next),
+                        next => checkRestoration(toLoc, cloudServer, versionId, next),
```

Comment:
restoration checks only added to versioned objects as unversioned object metadata is not updated correctly after restore-object is issued.